packages/cozy-pouch-link/src/CozyPouchLink.spec.js

```diff
@@ -124,6 +124,42 @@ describe('CozyPouchLink', () => {
         expect(true).toBe(true)
       })
     })
+
+    it('test if supportsOperation return false if the doctype is sync with a read only strategy', async () => {
```

Comment:
```suggestion
    test('supportsOperation returns false if the doctype is sync with a read only strategy', async () => {
```

`it` and `test` are named so that we can easily read what the test is about, let's use the correct one here ;)